packages/repl/src/lib/workers/bundler/index.js

```diff
@@ -339,8 +340,7 @@ async function get_bundle(uid, mode, cache, lookup) {
 }
 
 async function bundle({ uid, components }) {
-	if (import.meta.env.PROD) {
-		console.clear();
+	if (BROWSER) {
```

Comment:
should this be `!DEV`? I haven't looked at what it's actually doing, but that would match what it was before

Reply:
Oops. Mistake 😅